pom.xml

```diff
@@ -1463,6 +1463,9 @@
 					<printSummary>true</printSummary>
 					<runOrder>alphabetical</runOrder>
 					<failIfNoTests>false</failIfNoTests>
+					<forkCount>3</forkCount>
+                    <reuseForks>true</reuseForks>
+                    <argLine>-Xmx1024m</argLine>
```

Comment:
The indentation here suggests that lines 1467 and 1468 are indented with spaces, whereas the rest of the file is indented with tabs.  I despise tabs (because they cause exactly this sort of problem) but for consistency the spaces here should be swapped for tabs.

Reply:
don't know why, suppose my Eclipse has been set to replace all tab with spaces. Let me check.

Reply:
Maybe we should consider reformatting the entire pom.xml to remove the tabs.  It'd be spammy, but it has to be done eventually.  I think Nolan's IDE was set to prefer tabs at one point and that's what reformatted several files this way.

Reply:
4-space indent or 2-space indent? :)

Reply:
I'm a two-space guy insofar as XML goes, but four would be acceptable.  As far as I recall, Checkstyle can't enforce 2-vs-4 space indenting, but it definitely can ban tabs from occurring in source code.  ✊